install/kubernetes/tetragon/values.yaml

```
@@ -392,3 +392,8 @@ rthooks:
     override: ~
     repository: quay.io/cilium/tetragon-rthooks
     tag: v0.2
+  # -- rthooks service account.
+  serviceAccount:
+    create: false
+    annotations: {}
+    name: ""
```

To imitate the others SA you can do (and that's evolutive)
```suggestion
  serviceAccount:
    name: ""
```
Or, if we never plan to add create and annotations (maybe not a good idea)
```suggestion
  serviceAccountName: ""
```